lib/badges.js

```diff
@@ -1,127 +1,103 @@
 const crypto = require('crypto')
-
```

Comment:
Please do whitespace-only changes in a separate commit.

This allows us to skim over whitespace-only changes in larger functions (see below)